docs/develop/create-sso-office-add-ins-nodejs.md

```diff
@@ -128,449 +129,454 @@ This article walks you through the process of enabling single sign-on (SSO) in a
+
+   ```javascript
+   try {
+     // The access token returned from getAccessToken only has permissions to your middle-tier server APIs,
+     // and it contains the identity claims of the signed-in user.
+
+     const accessToken = await Office.auth.getAccessToken(authOptions);
+     return accessToken;
+   } catch (error) {
+     let fallbackRequired = handleSSOErrors(error);
+     if (fallbackRequired) throw error; // Rethrow the error and caller will switch to fallback auth.
+     return null; // Returning a null token indicates no need for fallback (an explanation about the error condition was shown by handleSSOErrors).
+   }
+   ```
+
+1. In the `handleSSOErrors` function, replace `TODO 6` with the following code. For more information about these errors, see [Troubleshoot SSO in Office Add-ins](troubleshoot-sso-in-office-add-ins.md).
+
+   ```javascript
+   let fallbackRequired = false;
+   switch (err.code) {
+   case 13001:
+     // No one is signed into Office. If the add-in cannot be effectively used when no one
+     // is logged into Office, then the first call of getAccessToken should pass the
+     // `allowSignInPrompt: true` option. Since this sample does that, you should not see
+     // this error.
+     showMessage(
+       "No one is signed into Office. But you can use many of the add-ins functions anyway. If you want to log in, press the Get OneDrive File Names button again."
+     );
+     break;
+   case 13002:
+     // The user aborted the consent prompt. If the add-in cannot be effectively used when consent
+     // has not been granted, then the first call of getAccessToken should pass the `allowConsentPrompt: true` option.
+     showMessage(
+       "You can use many of the add-ins functions even though you have not granted consent. If you want to grant consent, press the Get OneDrive File Names button again."
+     );
+     break;
+   case 13006:
+     // Only seen in Office on the web.
+     showMessage(
+       "Office on the web is experiencing a problem. Please sign out of Office, close the browser, and then start again."
+     );
+     break;
+   case 13008:
+     // Only seen in Office on the web.
+     showMessage(
+       "Office is still working on the last operation. When it completes, try this operation again."
+     );
+     break;
+   case 13010:
+     // Only seen in Office on the web.
+       showMessage(
+         "Follow the instructions to change your browser's zone configuration."
+       );
+       break;
+   ```
+
+1. Replace `TODO 7` with the following code. For more information about these errors, see [Troubleshoot SSO in Office Add-ins](troubleshoot-sso-in-office-add-ins.md). For any errors that can't be handled `true` is returned to the caller. This indicates the caller should switch to using MSAL as fallback auth.
```

Comment:
```suggestion
1. Replace `TODO 7` with the following code. For more information about these errors, see [Troubleshoot SSO in Office Add-ins](troubleshoot-sso-in-office-add-ins.md). For any errors that can't be handled `TRUE` is returned to the caller. This indicates the caller should switch to using MSAL as fallback auth.
```
TRUE and FALSE should be capitalized when referred to as arguments in your copy. OK to leave lowercase in code samples. This is for accessibility and ease of reading.

Reply:
Based on offline conversation we'll keep using the JS syntax.